client/marketing/overview/installed-extensions/style.scss

```diff
@@ -22,7 +18,7 @@
 	}
 
 	&__item:not(:last-child) {
-		border-bottom: 1px solid $gray-100;
+		border-bottom: 1px solid rgb(226, 228, 231);
```

Comment:
Is there a CSS variable you could use here instead of hardcoding the RGB?

Reply:
@becdetat , I end up using `$gray-200` from `@wordpress/base-styles`, which is meant for "light borders" and is also the closest to the desired border color.